static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -200,6 +200,11 @@ define([
                 }
             }
         }));
+        events.addContentEvents(player, mediaId, mediaType);
+        events.addPrerollEvents(player, mediaId, mediaType);
+        if (window.location.hash === '#debugVideoEvents') {
```

Comment:
would require a server restart for changes to be picked up, but would behave more like the `useHashedBundles` flag?